2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_setup.py:_flush():71] setting env: {}
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_120148-q878spfi/logs/debug.log
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_120148-q878spfi/logs/debug-internal.log
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_init.py:init():404] calling init triggers
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-06 12:01:48,466 INFO    MainThread:696295 [wandb_init.py:init():460] starting backend
2022-03-06 12:01:48,466 INFO    MainThread:696295 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-06 12:01:48,473 INFO    MainThread:696295 [backend.py:ensure_launched():216] starting backend process...
2022-03-06 12:01:48,476 INFO    MainThread:696295 [backend.py:ensure_launched():222] started backend process with pid: 696304
2022-03-06 12:01:48,477 INFO    MainThread:696295 [wandb_init.py:init():469] backend started and connected
2022-03-06 12:01:48,478 INFO    MainThread:696295 [wandb_init.py:init():533] updated telemetry
2022-03-06 12:01:48,490 INFO    MainThread:696295 [wandb_init.py:init():563] communicating current version
2022-03-06 12:01:49,488 INFO    MainThread:696295 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-06 12:01:49,489 INFO    MainThread:696295 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-06 12:01:49,900 INFO    MainThread:696295 [wandb_init.py:init():606] starting run threads in backend
2022-03-06 12:01:52,532 INFO    MainThread:696295 [wandb_run.py:_console_start():1810] atexit reg
2022-03-06 12:01:52,532 INFO    MainThread:696295 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-06 12:01:52,532 INFO    MainThread:696295 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-06 12:01:52,533 INFO    MainThread:696295 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-06 12:01:52,533 INFO    MainThread:696295 [wandb_init.py:init():633] run started, returning control to user process
2022-03-06 12:01:54,743 INFO    MainThread:696295 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-06 12:01:54,744 INFO    MainThread:696295 [wandb_run.py:_restore():1752] restore
2022-03-06 12:01:57,551 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 12:01:58,796 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 12:01:58,913 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 29837
}

2022-03-06 12:01:59,017 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 39436
}

2022-03-06 12:01:59,120 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 39436
}

2022-03-06 12:01:59,222 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 39436
}

2022-03-06 12:01:59,324 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:01:59,426 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:01:59,529 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:01:59,631 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:01:59,733 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:02:00,144 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}

2022-03-06 12:02:01,009 INFO    MainThread:696295 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39436
  total_bytes: 39436
}
local_info {
}

2022-03-06 12:02:02,074 INFO    MainThread:696295 [wandb_run.py:_append_files():2180] logging synced files
